src/main/java/service/impl/FruitRecordParserServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package service.impl;
+
+import exception.InvalidInputException;
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitRecordDto;
+import service.FruitRecordParserService;
+import service.validator.DataValidator;
+
+public class FruitRecordParserServiceImpl implements FruitRecordParserService {
+    public static final String COMMA = ",";
```

Comment:
```suggestion
    public static final String CSV_PARSER = ",";
```
name variables according to their purpose not the value they hold